src/openapi/spec/api-request-schema.ts

```diff
@@ -70,14 +70,42 @@ const schema = {
                                 parameters: {
                                     type: 'object',
                                 },
+                                variants: {
+                                    type: 'array',
+                                    items: {
+                                        type: 'object',
+                                        required: ['name', 'weight'],
+                                        additionalProperties: false,
+                                        properties: {
+                                            name: {
+                                                type: 'string',
+                                            },
+                                            weight: {
+                                                type: 'number',
+                                            },
+                                            stickiness: {
+                                                type: 'string',
+                                            },
+                                            payload: {
+                                                type: 'object',
+                                                additionalProperties: false,
+                                                required: ['type', 'value'],
+                                                properties: {
+                                                    type: { type: 'string' },
+                                                    value: { type: 'string' },
+                                                },
+                                            },
+                                        },
+                                    },
+                                },
                             },
                         },
                     },
                     variants: {
                         type: 'array',
                         items: {
                             type: 'object',
-                            required: ['name', 'weight', 'overrides'],
+                            required: ['name', 'weight'],
```

Comment:
node type doesn't expect overrides to always be present so adjusting this schema to fix TS errors